Makefile

```diff
@@ -12,6 +12,7 @@ GOBINDATA_BIN=$(GOPATH)/bin/go-bindata
 ENVVAR=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOOS=linux
 GO_BUILD_RECIPE=GOOS=$(GOOS) go build -o $(BIN) $(MAIN_PACKAGE)
+GOCACHE ?= off
```

Comment:
I think disabling test caching for integration/extended/e2e tests absolutely makes sense (otherwise the flake detection aspect is lost); however, I'm not sure why we'd want to disable test caching for unit tests.

Reply:
I ran 'make test/test-integration' multiple times and the tests ran once and subsequent test results came from cache. Now I think this should be the expected behavior as there is no change in code. Any code change is going to invalidate the cache. I will remove the GOCACHE env.